### PR TITLE
Print help when run 'pem' with a command

### PR DESCRIPTION
**Pull request by ivan-podorozhnyi**
Issue fixed, now result is the same as 'pem help'
Affects #37

### CODE REVIEW

File: bin/pem [hunk 1]
```diff
@@ -29,9 +29,8 @@ commands() {
 
 main() {
     local COMMAND="$1"; shift
-    [ -z "$COMMAND" ] && usage && exit 1
     [ "$COMMAND" = "commands" ] && commands && exit 0
-    [ ! -e "$PEM_DIR/pem-$COMMAND" ] && echo "Unknown command: pem $COMMAND" && usage && exit 2
+    [ ! -e "$PEM_DIR/pem-$COMMAND" -o -z "$COMMAND" ] && echo "Unknown command: pem $COMMAND" && usage && exit 2
```
Comment (extsoft) [inline bin/pem]: @ivan-podorozhnyi 

1. The issue is still reproducible. 
2. When call `pem`, we have to see `usage` output only.

**Review by extsoft — CHANGES_REQUESTED** [review]
@ivan-podorozhnyi , Take a look for debug output:
```
~/pem/bin export PEM_DEBUG=yes
~/pem/bin ./pem
++ dirname ./pem
+ PEM_DIR=.
+ main
+ local COMMAND=
+ shift
```

**Comment by ivan-podorozhnyi** [comment]
@extsoft please check updates. Seems now its working correctly

File: bin/pem [hunk 2]
```diff
@@ -28,8 +28,11 @@ commands() {
 }
 
 main() {
-    local COMMAND="$1"; shift
-    [ -z "$COMMAND" ] && usage && exit 1
+    local COMMAND="default"
+    if [ "$1" != "" ]; then
```
Comment (extsoft) [inline bin/pem:32]: @ivan-podorozhnyi use `-z` to check is var empty or not next time.

**Review by extsoft — APPROVED** [review]
@ivan-podorozhnyi good job!

**Pull request merged**